include/alpaka/dev/DevCpu.hpp

```diff
@@ -462,51 +463,4 @@ namespace alpaka
             }
         }
     }
-    namespace wait
```

Comment:
**Only a note:** moving this code to a new file and add the new feature that all include are verified make the review hard. Two pull request helps.